tests/utility/test_ip_rule.py

```diff
@@ -20,6 +20,12 @@ class TestIPRule:
     rule = Rule(trait='host.ip.address', action=RuleAction.DENY, match=subnet1)
     rs = RuleSet(rules=[rule])
 
+    def test_rule_display(self):
+        rule_display = self.rule.display
+        assert rule_display['trait'] == 'host.ip.address'
+        assert rule_display['action'] == 'DENY'
+        assert rule_display['match'] == self.subnet1
+
```

Comment:
Since we are testing rule serialization, should we test deserialization as well to cover both halves of the change?

Reply:
added an additional test for deserialization.